#bias: [2.7910467245312356, -12.146597830316438, 0.5177137360555685, -0.4192953713020189,
#  -0.37978352429582235, -0.014451804181081874]
#gripper_com_frame_id: l_force_torque_link
#gripper_com_pose: [0.0009781167696790768, 0.0020373140097300524, -0.08490294425172419,
# 0.0, 0.0, 0.0]
#gripper_mass: 1.0506110709852083

# May 15, 2017
bias: [2.9929410140799244, -12.935497150454092, 3.749019752854616, -0.4104847612395638,
  -0.3416986100638945, 0.0038532646007416183]
gripper_com_frame_id: l_force_torque_link
gripper_com_pose: [0.0013540552612281358, 0.0025828689463905687, -0.08485472347601578,
  0.0, 0.0, 0.0]
gripper_mass: 1.0601482762565713
